### PR TITLE
Package name in output

### LINKED ISSUE #103: Support printing package name for mono repos
Hi!  I have a mono repo with several packages inside.  When running `weeder` in project root it prints locations without a package info.  Consider we have
```
packages
  - pacakageA
    - src
      - Common.hs
      - … 
  - pacakageB
    - src
      - Common.hs
      - …
project.cabal
…
```
If both Common modules have unused functions weeder will output something like
```
src/Common.hs:111: unusedFuncInPackageA
src/Common.hs:123: unusedFuncInPackageB
```

Is it possible to make weeder to provide a package name?  Or maybe a flag to enable printing absolute file paths?  The latter may be very useful in VSCode for example since it supports jumping to location by Ctrl+clicking on paths in integrated terminal.

It looks like a trivial change and I would be glad to make a PR if someone provide a guidance or details how to implement this.

Thanks.

### PR DESCRIPTION
**Pull request by NorfairKing**
This goes on top of #155, fixes #103

### CODE REVIEW

**Comment by NorfairKing** [comment]
I couldn't figure out how to update the tests. Can you help me with that @ocharles ?

File: weeder.cabal [hunk 1]
```diff
@@ -78,6 +78,8 @@ test-suite weeder-test
   type: exitcode-stdio-1.0
   main-is: Spec.hs
   hs-source-dirs: test
+  build-tool-depends:
+    hspec-discover:hspec-discover
```
Comment (NorfairKing) [inline weeder.cabal:82]: This was necessary to get nix to put `hspec-discover` on the path for `cabal test` to run.

**Comment by ocharles** [comment]
> I couldn't figure out how to update the tests. Can you help me with that @ocharles ?

Can you try rebasing on https://github.com/ocharles/weeder/pull/159? Then you can run:

```
cabal run test:weeder-test -- --accept
```

**Comment by NorfairKing** [comment]
@ocharles Are you still around at Zürihack? I can't find you but will probably need a quick chat. I'm on the first floor.

**Comment by ocharles** [comment]
Hey, I think I was out for dinner then. I'm gonna be in later today, maybe try and grab me around 1? You can ping me on Discord if that's easier (ocharles_)

**Pull request merged**